### DISCUSSION TITLE
Error handling of API and managing internet connection lost

### DISCUSSION THREAD

**Discussion by bhargavraviya** [nylo-core/nylo#76] [Ideas] [answered]
- API Error Handling using Models Showing API Not Working
- Internet connection check
- Error Handling with proper models or dialog with showing error

@agordn52

**Comment by agordn52** [comment, marked as answer]
Hi @bhargavraviya,

A lot of this already exists, check out the [docs](https://nylo.dev/docs/5.x/networking) 👍
